tests/util/golinter/testdata/e2e/e2e_test.go

```diff
@@ -0,0 +1,68 @@
+package testdata
```

Comment:
Just curious.

is go test ... in the root directory going to run this too? =)

Reply:
That is possible. I just make the test work.